Packs/SecurityIntelligenceServicesFeed/Integrations/SecurityIntelligenceServicesFeed/SecurityIntelligenceServicesFeed.yml

```diff
@@ -70,13 +70,18 @@ configuration:
   required: false
 - display: Feed Fetch Interval
   name: feedFetchInterval
-  defaultvalue: "60"
+  defaultvalue: "5"
   type: 19
   required: false
+- display: Max Indicators Per Interval
```

Comment:
please add feedtags param to the yml and implementation in the code:
- additionalinfo: Supports CSV values.
  display: Tags
  name: feedTags
  required: false
  type: 0
you can see this pr for ref:
https://github.com/demisto/content/pull/8307

Reply:
Thanks, @orhovy Will add that.